docs/src/main/sphinx/admin/openmetrics.md

```diff
@@ -0,0 +1,134 @@
+# Cloud-scale Trino metrics with OpenMetrics
+
+Trino supports the metrics standard [OpenMetrics](https://openmetrics.io/), that
+originated with the open-source systems monitoring and alerting toolkit
+[Prometheus](https://prometheus.io/).
+
+Metrics are automatically enabled and available on the coordinator at the
+`/metrics` endpoint. The endpoint is protected with the configured
+[authentication](security-authentication), identical to the
+[](/admin/web-interface) and the [](/develop/client-protocol).  
+
+For example, you can retrieve metrics data from an unsecured Trino server
+running on `localhost:8080` with random username `example`:
+
+```shell
+curl -H X-Trino-User:foo localhost:8080/metrics
+```
+
+The result follows the [OpenMetrics
+specification](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md)
+and looks similar to the following example output:
+
+```
+# TYPE io_airlift_http_client_type_HttpClient_name_ForDiscoveryClient_CurrentResponseProcessTime_Min gauge
+io_airlift_http_client_type_HttpClient_name_ForDiscoveryClient_CurrentResponseProcessTime_Min NaN
+# TYPE io_airlift_http_client_type_HttpClient_name_ForDiscoveryClient_CurrentResponseProcessTime_P25 gauge
+io_airlift_http_client_type_HttpClient_name_ForDiscoveryClient_CurrentResponseProcessTime_P25 NaN
+# TYPE io_airlift_http_client_type_HttpClient_name_ForDiscoveryClient_CurrentResponseProcessTime_Total gauge
+io_airlift_http_client_type_HttpClient_name_ForDiscoveryClient_CurrentResponseProcessTime_Total 0.0
+# TYPE io_airlift_http_client_type_HttpClient_name_ForDiscoveryClient_CurrentResponseProcessTime_P90 gauge
+io_airlift_http_client_type_HttpClient_name_ForDiscoveryClient_CurrentResponseProcessTime_P90 NaN
+```
+
+The same data is available when using a browser, and logging manually.
+
+Typically, Prometheus or a similar application is configured to monitor the
+endpoint. The same application can then be used to inspect the metrics data.
+
+Trino also includes a [](/connector/prometheus) that allows you to query
+Prometheus data using SQL.
+
+## Example use
```

Comment:
i really think that running  Prometheus server is out of scope for Trino Documentation